email-alert-api/docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+version: '3.7'
+
+x-email-alert-api: &email-alert-api
+  build:
+    context: .
+    dockerfile: email-alert-api/Dockerfile
+  image: email-alert-api
+  volumes:
+    - ..:/govuk:delegated
+    - bundle:/usr/local/bundle
+  working_dir: /govuk/email-alert-api
+
+services:
+  email-alert-api-default:
+    <<: *email-alert-api
+    depends_on:
+      - postgres
+      - redis
+    environment:
+      DATABASE_URL: "postgresql://postgres@postgres/email-alert-api"
+      TEST_DATABASE_URL: "postgresql://postgres@postgres/email-alert-api-test"
+      REDIS_URL: redis://redis
+
+  email-alert-api-backend: &email-alert-api-backend
+    <<: *email-alert-api
+    depends_on:
+      - postgres
+      - redis
+    environment:
+      DATABASE_URL: "postgresql://postgres@postgres/email-alert-api"
+      TEST_DATABASE_URL: "postgresql://postgres@postgres/email-alert-api-test"
```

Comment:
Oh, forgot that we shouldn't need to set this in other stacks, since we don't need a test DB.

Reply:
Good catch